extension/content.js

```diff
@@ -263,6 +263,25 @@ function addPatchDiffLinks() {
 	`);
 }
 
+// Prompt user to confirm erasing a comment with the Cancel button
+$(document).on('click', function (event) {
+	// Check event.target instead of using a delegate, because Sprint doesn't support them
```

Comment:
I initially tried using vanilla JS, but it just got too verbose and cumbersome.

My question wasn't about keeping jQuery up to date though. Very little changes these days. But there are clearly missing features from Sprint.